smart-cards/src/main/java/gov/va/api/health/smartcards/patient/PatientController.java

```diff
@@ -228,4 +243,14 @@ private void lookupAndAttachLocations(Immunization.Bundle immunizations, String
               .collect(toList()));
     }
   }
+
+  @SneakyThrows
+  private String signVc(VerifiableCredential vc, String vcJws, String vcCompress) {
```

Comment:
`signVc(VerifiableCredential vc, boolean shouldSign, boolean shouldCompress)` is a better signature for this method. Convert the strings to booleans back in the entry-point method.